### PR TITLE
postrm: remove log files on package purge

### LINKED ISSUE #586: Remove log files on purge
According to https://www.debian.org/doc/debian-policy/ch-files.html#log-files, log files should be removed when the package is purged:

> Log files should be removed when the package is purged (but not when it is only removed). This should be done by the postrm script when it is called with the argument purge (see Details of removal and/or configuration purging).

### PR DESCRIPTION
**Pull request by panlinux**
If purging, remove the log file(s). Globbing there is to catch rotated and compressed or not logs.

Fixes #586

### CODE REVIEW

File: debian/postrm [hunk 1]
```diff
@@ -15,6 +15,10 @@ remove_cache_dir(){
     rm -rf /var/lib/ubuntu-advantage
 }
 
+remove_logs(){
+    rm -f /var/log/ubuntu-advantage.log*
```
Comment (blackboxsw) [inline debian/postrm]: Why the asterisk?

Comment (OddBloke) [reply debian/postrm]: Log rotation?

Comment (panlinux) [reply debian/postrm]: Globbing there is to catch rotated and compressed or not logs.

Comment (powersj) [reply debian/postrm]: log rotate right?

**Pull request merged**